core/domain/classifier_services.py

```diff
@@ -214,6 +214,13 @@ def save_classifier(classifier):
         classifier_id = _create_classifier(classifier)
     else:
         _update_classifier(classifier_data_model, classifier.state_name)
+
+    # Create the mapping from <exp_id,exp_version,state_name>
+    # to classifier_id.
+    create_classifier_exploration_mapping(
```

Comment:
The docstring for this function says that there's a case where the model might already exist. Does that case actually occur in practice? If so, in such a case, do we still need to create this mapping?